common/log/tag/tags.go

```diff
@@ -56,6 +60,11 @@ func Error(err error) ZapTag {
 	return NewErrorTag(err)
 }
 
+// ErrorType returns tag for ErrorType
+func ErrorType(err error) ZapTag {
```

Comment:
can we have some unit test just to verify the string looks reasonable for a few error types.